octal/cavium_env/rf_card_cal_tip/test_config.txt

```diff
@@ -14,8 +14,9 @@ dl_freq_5=875                       # downlink frequency
 ul_freq_5=830                       # uplink frequency
 
 band_28=28                         # enodeB band
+#dl_freq_28=780.5                       # downlink frequency
```

Comment:
Since ul_freq is at center band, do you want the dl_freq to be at center band too? Since this is a config file, it will change from user to user, so no worries if this is your intention

Reply:
good point, it should have been the 780.5 , the ENS system sets txfreq to 790 and i forgot to switch it back in the check in.  switching now